pkg/controllers/vdb/podfacts.go

```diff
@@ -399,6 +410,8 @@ func (p *PodFacts) genGatherScript(vdb *vapi.VerticaDB, pf *PodFact) string {
 		df --block-size=1 --output=avail %s | tail -1
 		echo -n 'agentRunning: '
 		/opt/vertica/sbin/vertica_agent status | grep --quiet "running" && echo true || echo false
+		echo -n 'hasAgentKeys: '
+		test -f %s/agent.key && test -f %s/agent.cert && test -f %s/apikeys.dat && echo true || echo false
```

Comment:
I wonder if we should check if the directory is empty. Is there ever a case where only some keys are present? There is an example of checking for an empty dir above in the `dbExists` check.

Reply:
We want all keys to the here so what will be the benefit of checking if the dir is empty?

Reply:
We want all keys to the here so what will be the benefit of checking if the dir is empty?

Reply:
That way you don't have to spell out all of the keys you want. It makes it easier in case there are other things added in here in the future. You are basically just checking to see if the image that we are using preserved agent artifacts that need to be copied over to /opt/vertica/confg. So, why do we care that specific files are there.

Reply:
Yes there are no cases where only some of them are here.